src/DssSpell.sol

```diff
@@ -48,36 +151,275 @@ contract DssSpellAction is DssAction {
+        // Init new Oracle with wrapper_ parameter being 0x38e8c1D443f546Dc014D7756ec63116161CB7B25
+        // Init new Oracle with divisor parameter being 24,000
+        // Init new Oracle with clKey parameter being FLAP_SKY_ORACLE
+        FlapperInit.initOracleWrapper(
+            dss,
+            FLAP_SKY_ORACLE,
+            24_000,
+            "FLAP_SKY_ORACLE"
+        );
+
+
+        // ---------- Setup DssVestMintable for SKY ----------
+        // Forum: TODO
+        // Poll: TODO
+        // MIP: TODO
+
+        // Authorize DssVestMintable on SKY by calling DssExecLib.authorize with the following parameters:
+        // Authorize DssVestMintable on SKY with _base parameter being 0x56072C95FAA701256059aa122697B133aDEd9279
+        // Authorize DssVestMintable on SKY with _ward parameter being 0xB313Eab3FdE99B2bB4bA9750C2DDFBe2729d1cE9
+        DssExecLib.authorize(SKY, MCD_VEST_SKY);
+
+        // Set DssVestMintable max rate (cap) by calling DssExecLib.setValue with the following parameters:
+        // Set DssVestMintable max rate (cap) with _base parameter being 0xB313Eab3FdE99B2bB4bA9750C2DDFBe2729d1cE9
+        // Set DssVestMintable max rate (cap) with _what parameter being "cap"
+        // Set DssVestMintable max rate (cap) with _amt parameter being 799,999,999.999999999985808000 Sky per year (800M * WAD / 365 days )
+        DssExecLib.setValue(MCD_VEST_SKY, "cap", 800 * MILLION * WAD / 365 days);
+
+        // Add DssVestMintable to Chainlog by calling DssExecLib.setChangelogAddress with the following parameters:
+        // Add DssVestMintable to Chainlog with _key parameter being MCD_VEST_SKY
+        // Add DssVestMintable to Chainlog with _val parameter being 0xB313Eab3FdE99B2bB4bA9750C2DDFBe2729d1cE9
+        DssExecLib.setChangelogAddress("MCD_VEST_SKY", MCD_VEST_SKY);
+
+        // ---------- USDS => SKY Farm Setup ----------
+
+        // Init USDS -> SKY rewards by calling UsdsSkyFarmingInit.init with the following parameters:
+        // Init USDS -> SKY rewards with usds parameter being 0xdC035D45d973E3EC169d2276DDab16f1e407384F
+        // Init USDS -> SKY rewards with sky parameter being 0x56072C95FAA701256059aa122697B133aDEd9279
+        // Init USDS -> SKY rewards with rewards parameter being 0x0650CAF159C5A49f711e8169D4336ECB9b950275
+        // Init USDS -> SKY rewards with rewardsKey parameter being REWARDS_USDS_SKY
+        // Init USDS -> SKY rewards with dist parameter being 0x2F0C88e935Db5A60DDA73b0B4EAEef55883896d9
+        // Init USDS -> SKY rewards with distKey parameter being REWARDS_DIST_USDS_SKY
+        // Init USDS -> SKY rewards with vest parameter being 0xB313Eab3FdE99B2bB4bA9750C2DDFBe2729d1cE9
+        // Init USDS -> SKY rewards with vestTot parameter being 600M * WAD
+        // Init USDS -> SKY rewards with vestBgn parameter being block.timestamp - 7 days
+        // Init USDS -> SKY rewards with vestTau parameter being 365 days
```

Comment:
The [instruction in the Exec Sheet](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?gid=1165645248#gid=1165645248&range=B129) now says `365 - 1 days`